soccer/planning/planner/PivotPathPlanner.cpp

```diff
@@ -71,24 +75,33 @@ Trajectory PivotPathPlanner::plan(const PlanRequest& request) {
         points.push_back(point);
     }
 
-    BezierPath pathBezier(points, start_instant.linear_velocity(), Point(0, 0),
+    BezierPath pathBezier(points, Point(0, 0), Point(0, 0),
                           linear_constraints);
 
     Trajectory path =
         ProfileVelocity(pathBezier, start_instant.linear_velocity().mag(), 0,
                         linear_constraints, start_instant.stamp);
+    {
+        // Stay in place
+        RobotInstant instant = path.last();
+        instant.stamp = instant.stamp + RJ::Seconds(5.0);
+        path.AppendInstant(instant);
+    }
```

Comment:
This might be a visual glitch because I'm on mobile. These braces seem useless?

Reply:
They're useless, I like to do this to make a temporary that shouldn't be used anywhere else. It's just personal preference, I can get rid of it if you'd rather not have it.

Reply:
I gotcha. Personally I'm not a fan of it because it messes with the tabbing whitespace, but I'm also not doing much coding anymore sooooo it's up to you guys

Reply:
I think its a bit of visual clutter. I'm not a big fan of it either

Reply:
Fixed in #1542 by deleting this code anyway, lmao